.github/workflows/zscaler-iac-scan.yml

```
@@ -36,21 +36,22 @@ jobs:
+          # iac_file : #Enter the IaC file path from root.
+          # output_format : #(Optional) By default, the output is provided in a human readable format. However, if you require a different format, you can specify it here.
           #To fail the build based on policy violations identified in the IaC templates, set the input value (fail_build) to true.
-          fail_build : #Enter true/false
+          # fail_build : #Enter true/false
       #Ensure that the following step is included in order to post the scan results under the code scanning alerts section within the repository.
       - name: Upload SARIF file
-        if: ${{ success() || failure() && (steps.zscaler-iac-scan.outputs.sarif_file_path != '') }}
+        if: ${{ (success() || failure()) && steps.zscaler-iac-scan.outputs.sarif_file_path != '' }}
         uses: github/codeql-action/upload-sarif@v3
         with:
           sarif_file: ${{ steps.zscaler-iac-scan.sarif_file_path }}
```

actionlint (1.7.11)

[error] property "sarif_file_path" is not defined in object type {conclusion: string; outcome: string; outputs: {string => string}}

(expression)